### PR TITLE
template for requesting addition of a new species

### PR DESCRIPTION
**Pull request by petrelharp**
This would be a way for someone to compile the relevant information for adding a new species without doing the coding.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-species-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+**Demographic information:**
+
+- [] generation time in years
+- [] "default" population size
+
+
+**Chromosome structure:**
+
+- [] list of chromosomes with *name* and *length* (in bp)
```
Comment (grahamgower) [inline .github/ISSUE_TEMPLATE/new-species-issue-template.md:36]: These are coming from ensembl now, so this isn't strictly necessary, right?

**Review by grahamgower — APPROVED** [review]
LGTM. Maybe we don't need the chromosome list though?

File: .github/ISSUE_TEMPLATE/new-species-issue-template.md (hunk 1 above)
Comment (jeromekelleher) [reply .github/ISSUE_TEMPLATE/new-species-issue-template.md:36]: Speaking of which, the Ensembl ID is a necessary bit of info.

File: .github/ISSUE_TEMPLATE/new-species-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+**Recombination rates:**
+
+- [] genetic map (as a .csv) of recombination rates **(optional)**
```
Comment (jeromekelleher) [inline .github/ISSUE_TEMPLATE/new-species-issue-template.md:41]: Or Hapmap? Don't want to people to convert to CSV if it's already in the right format.

**Review by jeromekelleher — APPROVED** [review]
LGTM, but we should give more info on how to get the Ensembl ID, etc.

**Comment by petrelharp** [comment]
> Speaking of which, the Ensembl ID is a necessary bit of info.

Hm. So, here's the species that are apparently available on ensembl:

- [mostly mammals](https://uswest.ensembl.org/info/about/species.html)
- ["metazoa"](https://metazoa.ensembl.org/species.html)
- [plants](https://plants.ensembl.org/species.html)
- [fungi](https://fungi.ensembl.org/species.html)
- [protists](http://protists.ensembl.org/species.html)
- [bacteria](http://bacteria.ensembl.org/species.html)

I think that's all? I can't find a list of ensembl sites.

And, what do we do about species not in Ensembl (e.g., there are no Mimulus, looks like)? Do we say that we're only for species with annotations uploaded to Ensembl? Any idea how difficult getting a new species on there is?

**Comment by jeromekelleher** [comment]
I guess we could put the data files in by hand for species that aren't in Ensembl. We'd have to introduce some level of QC then, though.

> Any idea how difficult getting a new species on there is?

No - can't imagine it's a quick process though, by the time the data gets into a release.

**Comment by petrelharp** [comment]
> I guess we could put the data files in by hand for species that aren't in Ensembl. We'd have to introduce some level of QC then, though.

Can we do this currently or would this require a lot of changes to the infrastructure? In other words, is this something we want to allow in the "adding to the zoo" workshop?

**Comment by petrelharp** [comment]
Ok, I've updated this - I think it's OK now, unless we think we will never want to support non-Ensembl species.

**Comment by petrelharp** [comment]
More Ensembl info: from [this page](https://uswest.ensembl.org/info/genome/genebuild/assembly.html)
> Ensembl does not produce genome assemblies, instead we provide annotation on genome assemblies that have been deposited into the [INSDC](http://www.ebi.ac.uk/ena) (GenBank, ENA, DDBJ) and are publicly available. We select species to annotate on a case-by-case basis ...

**Comment by jeromekelleher** [comment]
> Can we do this currently or would this require a lot of changes to the infrastructure? In other words, is this something we want to allow in the "adding to the zoo" workshop?

No, probably not. I think we should focus on Ensembl species for the workshop.

**Review by jeromekelleher — APPROVED** [review]
LGTM!

**Pull request merged**